siemens/questa.mk

```diff
@@ -158,9 +177,16 @@ $(DEP_DIR)/%.questa.o: $(SIM_LIB_DONE) | $(DEP_DIR) $(BLOG_DIR)
 	else echo "Unknown filetype: $(word 2,$^)"; echo "$^"; exit 1; fi; fi;
 	@touch $@
 
+AC_CMD := qverify -c -do $(AC_SCRIPT) -od $(AC_OUT_DIR) -modelsimini $(MS_INI)
+AC_MSG := Running Autocheck
+$(AC_DONE): $(MS_INI) $(DEP_DIR)/$(_TOP).questa.o $(precomp_hook) $(AC_DIRECTIVES)
+	@printf "$(autocheck_str)" > $(AC_SCRIPT)
+	@$(HDL_BUILD_PATH)/siemens/run_siemens.sh '$(AC_MSG)' '$(AC_CMD)' '$(BLOG_DIR)/autocheck.log';
+	@echo -e "$O Starting autocheck simulation $C (see $(BLOG_DIR)/autocheck.log)"
```

Comment:
This message is in the wrong place. I'm guessing it is superfluous to the `AC_MSG`. Maybe change `AC_MSG` to 
```
AC_MSG := $O Starting autocheck simulation $C (see $(BLOG_DIR)/autocheck.log)
```